app/jobs/regular/publish_group_membership_updates.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Jobs
+  class PublishGroupMembershipUpdates < ::Jobs::Base
+    def execute(args)
+      group = Group.find_by(id: args[:group_id])
+      users = User.human_users.where(id: args[:user_ids])
+
+      return if !group
+      return if !%w[add remove].include?(args[:type])
```

Comment:
This check can be moved up in the method. Also, I don’t think we should silently return here since it will hide any mistakes people make when enqueuing this job.